src/main/java/controller/ElementsController.java

```diff
@@ -0,0 +1,78 @@
+package controller;
+
+import javafx.collections.ObservableList;
+import javafx.scene.control.TextArea;
+import javafx.scene.control.TextField;
+
+import java.util.ArrayList;
+import java.util.Collections;
+
+/**
+ * Created by Maciek on 2017-02-10.
+ */
+public abstract class ElementsController {
+    public TextField adres_linku;
```

Comment:
ok czaje fieldsy sie overajduja z superklasy kurde ale magic nigdy tak nie robilem. Uwazasz ze to jest ladne ? :D teraz jak bym zrobil refaktor w jednej klasie zmeinil powiedzmy jedna nazwe a w dziedziczacej nie to sei wszystko sypnie xd